packages/react-native-bpk-component-chip/src/BpkChipWrapper.js

```diff
@@ -120,39 +148,52 @@ const BpkChipWrapper = (props: Props) => {
     iconStyle.push(styles.iconDisabled);
   }
 
+  if (type === CHIP_TYPES.outline) {
+    userStyle.push({ elevation: 0 });
+    innerStyle.push(styles.innerOutline);
+    textStyle.push(styles.textOutline);
+    iconStyle.push(styles.iconOutline);
+
+    if (selected) {
+      innerStyle.push(styles.innerOutlineSelected);
+      textStyle.push(styles.textSelected);
+    }
+    if (disabled) {
+      innerStyle.push(styles.innerOutlineDisabled);
+      textStyle.push(styles.textOutlineDisabled);
+      iconStyle.push(styles.iconOutlineDisabled);
+    }
+  }
+
   const themeAttributes = {
     ...getThemeAttributes(REQUIRED_THEME_ATTRIBUTES, theme),
-    ...getThemeAttributes(OPTIONAL_THEME_ATTRIBUTES, theme),
   };
 
   if (themeAttributes) {
-    if (themeAttributes.colorGray700 && themeAttributes.colorGray500) {
-      textStyle.push({
-        color: themeAttributes.colorGray700,
-      });
-      iconStyle.push({
-        color: themeAttributes.colorGray500,
-      });
-    }
-    if (
-      selected &&
-      themeAttributes.chipSelectedBackgroundColor &&
-      themeAttributes.chipSelectedTextColor
-    ) {
-      innerStyle.push({
-        backgroundColor: themeAttributes.chipSelectedBackgroundColor,
-      });
-      textStyle.push({
-        color: themeAttributes.chipSelectedTextColor,
-      });
-    }
-    if (disabled && themeAttributes.colorGray300) {
-      textStyle.push({
-        color: themeAttributes.colorGray300,
-      });
-      iconStyle.push({
-        color: themeAttributes.colorGray300,
-      });
+    if (selected && !disabled) {
+      if (
```

Comment:
Can we refactor this into a variable to avoid having so many nested `if`s?
Something like:
```
const shouldApplyTheme = themeAttributes && selected && !disabled;
const shouldApplyPrimaryTheme = shouldApplyTheme && type....;
const shouldApplyOutlineTheme = shouldApplyTheme && type....;
```

Reply:
done!